### PR TITLE
updating installation from source

### PR DESCRIPTION
**Pull request by lzap**
https://github.com/theforeman/theforeman.org/issues/90

### CODE REVIEW

File: _includes/manuals/1.2/3.4_install_from_source.md [hunk 1]
```diff
@@ -23,10 +23,12 @@ the database it uses and setup the necessary schema.
...
 git clone https://github.com/theforeman/foreman.git -b develop
 cd foreman
-# must have the bundler gem installed -- gem install bundle
-bundle install --without mysql mysql2  sqlite test --path vendor # or postgresql
 cp config/settings.yaml.example config/settings.yaml
 cp config/database.yml.example config/database.yml
+gem install bundler
+# depending on database configured in yml file you can skip others
+# (we are assuming sqlite to be configured)
+bundle install --without mysql mysql2 pg test --path vendor # or postgresql
```
Comment (domcleal) [inline _includes/manuals/1.2/3.4_install_from_source.md]: I think the bundler group is "postgresql"

Comment (domcleal) [inline _includes/manuals/1.2/3.4_install_from_source.md]: comment on the end of the line can be removed now?

Comment (lzap) [inline _includes/manuals/1.2/3.4_install_from_source.md]: Right fixed

On Fri, Sep 27, 2013 at 06:31:41AM -0700, Dominic Cleal wrote:

> >  cp config/settings.yaml.example config/settings.yaml
> >  cp config/database.yml.example config/database.yml
> > +gem install bundler
> > +# depending on database configured in yml file you can skip others
> > +# (we are assuming sqlite to be configured)
> > +bundle install --without mysql mysql2 pg test --path vendor # or postgresql
> 
> comment on the end of the line can be removed now?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/theforeman/theforeman.org/pull/91/files#r6627481

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

**Comment by domcleal** [comment]
Thanks @lzap!

**Pull request merged**